packages/cli/src/audit/risks/credentials.risk.ts

```diff
@@ -0,0 +1,133 @@
+import { MoreThanOrEqual } from 'typeorm';
+import { DateUtils } from 'typeorm/util/DateUtils';
+import * as Db from '@/Db';
+import config from '@/config';
+import { CREDENTIALS_REPORT } from '@/audit/constants';
+import type { WorkflowEntity } from '@/databases/entities/WorkflowEntity';
+import type { Risk } from '@/audit/types';
+
+async function getAllCredsInUse(workflows: WorkflowEntity[]) {
+	const credsInAnyUse = new Set<string>();
+	const credsInActiveUse = new Set<string>();
+
+	workflows.forEach((workflow) => {
+		workflow.nodes.forEach((node) => {
+			if (!node.credentials) return;
+
+			Object.values(node.credentials).forEach((cred) => {
+				if (!cred?.id) return;
+
+				credsInAnyUse.add(cred.id);
+
+				if (workflow.active) credsInActiveUse.add(cred.id);
+			});
+		});
+	});
+
+	return {
+		credsInAnyUse,
+		credsInActiveUse,
+	};
+}
+
+async function getAllExistingCreds() {
+	return Db.collections.Credentials.find().then((c) =>
+		c.map((i) => ({ kind: 'credential' as const, id: i.id, name: i.name })),
+	);
+}
```

Comment:
since this is already an async function why not use await?
```suggestion
async function getAllExistingCreds() {
	const credentials = await Db.collections.Credentials.find({ select: ['id', 'name'] });
	return credentials.map(({ id, name }) => ({ kind: 'credential' as const, id, name }));
}
```